certified-connectors/BiG EVAL/apiDefinition.swagger.json

```diff
@@ -0,0 +1,513 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "BiG EVAL",
+    "description": "BiG EVAL Data Quality Automation platform for data validation and data test automation.",

```

Comment:
@boltth description should contain information about the main value and purpose offered by the connector. Please make it more concise. You can refer to the documentation for more [information](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector).

Reply:
done